docs/src/main/asciidoc/ansible.adoc

```diff
@@ -0,0 +1,238 @@
+
+With those requirements in place, the jar can be deployed on the target, along with the required configuration for the app integration into systemd as a service. Any change to the systemd configuration requires reloading its daemon, which the collection ensures will happen whenever it is needed. With all of that in place, the only thing that remains is to start the service itself, which Ansible will also take care of.
+
+=== Validate that deployment was successful
+
+For the purpose of this tutorial, you may want to check manually, that the playbook did indeed deployed the app properly. Here is the few ways to do so.
+
+First, because the collection integrated, we can check the status of the service on one of the targets:
+
+[source,bash]
+----
+# systemctl status getting-started.service
+● getting-started.service - A Quarkus service named getting-started
+   Loaded: loaded (/usr/lib/systemd/system/getting-started.service; enabled; vendor preset: disabled)
+   Active: active (running) since Thu 2023-04-13 12:48:18 UTC; 2min 40s ago
+ Main PID: 853 (java)
+	Tasks: 43 (limit: 1638)
+   Memory: 73.3M
+   CGroup: /system.slice/getting-started.service
+       	└─853 /usr/bin/java -jar /opt/quarkus_deploy/quarkus-run.jar
+
+Apr 13 12:48:18 bd71f39642c8 systemd[1]: Started A Quarkus service named getting-started.
+Apr 13 12:48:19 bd71f39642c8 java[853]: __  ____  __  _____   ___  __ ____  ______
+Apr 13 12:48:19 bd71f39642c8 java[853]:  --/ __ \/ / / / _ | / _ \/ //_/ / / / __/
+Apr 13 12:48:19 bd71f39642c8 java[853]:  -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \
+Apr 13 12:48:19 bd71f39642c8 java[853]: --\___\_\____/_/ |_/_/|_/_/|_|\____/___/
+Apr 13 12:48:19 bd71f39642c8 java[853]: 2023-04-13 12:48:19,284 INFO  [io.quarkus] (main) getting-started 1.0.0-SNAPSHOT on JVM (powered by Quarkus 2.16.6.Final) started in 0.607s. Listening on: http://0.0.0.0:8080
+Apr 13 12:48:19 bd71f39642c8 java[853]: 2023-04-13 12:48:19,309 INFO  [io.quarkus] (main) Profile prod activated.
+Apr 13 12:48:19 bd71f39642c8 java[853]: 2023-04-13 12:48:19,310 INFO  [io.quarkus] (main) Installed features: [cdi, resteasy-reactive, smallrye-context-propagation, vertx]
+----
+
+Manually, you can also test if the app is reachable:
+
+[source,bash]
+----
+# curl -I http://localhost:8080/
+HTTP/1.1 200 OK
+accept-ranges: bytes
+content-length: 3918
+cache-control: public, immutable, max-age=86400
+last-modified: Thu, 2 Mar 2023 11:03:18 GMT
+date: Thu, 2 Mar 2023 11:03:18 GMT
+----
+
+We'll see how to automate those validation in the next and last part of this tutorial.
+
+=== Writing a playbook
+
+Of course, you’ll most likely need to build on this, so you may want to write up your own playbook, rather than just using the one provided by the collection. Below is an example of such playbook:
+
+[source,yml]
```

Comment:
I had problems copying this section. The tab characters generated a malformed `yaml` file that ansible complained when parsing. Please check this.

On the other hand, I had to declare the collection at the beginning of the playbook when used for the first time because Ansible was unable to identify the dependency on the `quarkus` role.

```yaml
- name: "Build and deploy a Quarkus app using Ansible"
...
  collections:
    - middleware_automation.quarkus
...
```

https://docs.ansible.com/ansible/latest/collections_guide/collections_using_playbooks.html#using-collections-in-playbooks

Reply:
Fixed (hopefully) in [commit 32860](https://github.com/quarkusio/quarkus/pull/32860/commits)